src/main/resources/Help/Code/Translations.xml

```diff
@@ -0,0 +1,227 @@
+help.macros.examples.description=Need help or confused with some of our macros? Check out their documentation and find out more about their functionality.
+help.macros.examples.title=Examples
+help.macros.gallery={0}Gallery Macro{1} displays a list of images in a slide-show gallery.
+help.macros.gallery.description=Displays a list of images in a slide-show gallery.
+help.macros.gallery.title=Gallery
+help.macros.header=Section
+help.macros.header1=Sub-Section
+help.macros.info=Info Macro
+help.macros.info.message=This is an info message
+help.macros.more=View more macros
+help.macros.notifications.activity.title1=Wiki Activity
+help.macros.notifications.activity.title2=Current User Activity
+help.macros.notifications.content={0}Notifications Macro{0} provides information about recent activities done by the users.
+help.macros.notifications.description=Displays recent activities done on pages by the users.
+help.macros.notifications.title=Notifications
+help.macros.success=Success Macro
+help.macros.success.message=This is a success message
+help.macros.title=Macros
+help.macros.toc={0}Table of Contents Macro{0} generates a table of contents for the page headings.
+help.macros.toc.description.content= The following examples use the current page structure constituted by two sections and three subsections. Learn more about structuring a page with headings {0}on the syntax page.{1}
+help.macros.toc.description=Generates a table of contents for the page headings.
+help.macros.toc.description.title=Description
+help.macros.toc.examples.title=Macro Table of Contents Examples
+help.macros.toc.insideBox=Table of Contents inside a box
+help.macros.toc.simple=Simple Table of Contents
+help.macros.toc.startingPoint=Table of Contents with a starting point
+help.macros.toc.title=Table of Contents
+help.macros.tree.description=Displays the hierarchy of pages.
+help.macros.tree.subtitle=Tree
+help.macros.tree.pageDescription={0}Page Tree Macro{0} displays the hierarchy of pages.
+help.macros.tree.title=Page Tree
+help.macros.tree.header1=Page Hierarchy Tree
+help.macros.tree.header2=Compact Page Hierarchy Tree
+help.macros.tree.header3=Wiki {0} Space {0} Page Tree
+help.macros.tree.header4=Full Entity Tree
+help.macros.useMacros.description=Macros are reusable pieces of functionality that can be integrated inside pages.
+help.macros.useMacros.tip=The {0}WYSIWYG Editor{1} allows you to insert macros and to customize their parameters when editing pages. Look for the {2} button on the tool bar.
+help.macros.useMacros.title=Use Macros
+help.macros.warning=Warning Macro
+help.macros.warning.message=This is a warning message
+help.more.label=Learn more
+help.organization.description=Organize your content in hierarchical pages, grouped in several (sub)wikis.
+help.organization.title=Content Organization
+help.start.description=Easy steps to get you started with XWiki.
+help.start.title=Getting Started
+help.templates.article.description=Article page
```

Comment:
I guess the help.templates.* translations are not required anymore since we directly linked to the real templates now.

Reply:
Not all of them should be deleted actually, I see some of them for the general explanation of XWiki templates.

Reply:
I cannot check in Github diff since the Templates.xml diff is too big, could you confirm @CamiAndrei that those translations are not used anymore, or are they for presenting the templates in cards?

Reply:
The keys containing help.templates.* are used only for presenting the templates in cards, the translation keys used in previous PR for Templates were removed.